CONTRIBUTING.md

```diff
@@ -170,3 +170,14 @@ We have a low-traffic mailing list for announcements of new `xrpl.js` releases.
 If you're using the XRP Ledger in production, you should run a [rippled server](https://github.com/ripple/rippled) and subscribe to the ripple-server mailing list as well.
 
 + [Subscribe to ripple-server](https://groups.google.com/g/ripple-server)
+
+# How to upgrade npm packages
```

Comment:
Can this upgrade change be done in a separate PR?

Reply:
Sure but I'd rather not as it's a low risk change

Reply:
Fixed